README.md

```diff
@@ -1 +1,24 @@
 # hw01
+
+# Adeline Kao Personal Biography
+## Introduction
+**Hello! My name is Adeline Kao and this is my short personal biography.**
+*(I go by the pronouns she, her, hers.)*
+## Personal Biography
+![Image of Adeline Kao](Adeline_Kao_Picture.png)
+
+I was born on August 13th, 1998 in Idaho.
+On November 28th, 2000, my younger brother, Kevin, was born.
+Before my eighth birthday, my family moved to Texas where I spent most of my life.
+While living in Idaho, I learned how to play the piano and when my family moved to Texas I learned how to play the violin.
+
+I am (unfortunately) allergic to:
+1. Cats
```

Comment:
Please try to use an unordered list next time (the numeric order implicates some ranks?)